### PR TITLE
Updating Component Assessment Module

### PR DESCRIPTION
**Pull request by gwaygenomics**
related to #94 but I also update the size of all plots in this module

### CODE REVIEW

File: 4.analyze-components/scripts/nbconverted/1.visualize-reconstruction.r [hunk 1]
```diff
@@ -18,10 +18,15 @@ base_dir <- file.path("figures", "TARGET")
```
Comment (ajlee21) [inline 4.analyze-components/scripts/nbconverted/1.visualize-reconstruction.r]: Substitute "dataset" variable here

File: 4.analyze-components/scripts/nbconverted/1.visualize-reconstruction.r [hunk 2]
```diff
@@ -18,10 +18,15 @@ base_dir <- file.path("figures", "TARGET")
 
 target_recon_gg <- plot_reconstruction_loss(target_recon_cost_df)
 
-ggsave(file.path(base_dir, paste0("reconstruction_cost_", dataset, ".pdf")),
-       plot = target_recon_gg, dpi = 500, height = 3, width = 9)
-ggsave(file.path(base_dir, paste0("reconstruction_cost_", dataset, ".png")),
-       plot = target_recon_gg, height = 3, width = 9)
+for(extension in c('.png', '.pdf')) {
```
Comment (ajlee21) [inline 4.analyze-components/scripts/nbconverted/1.visualize-reconstruction.r]: Why are some pngs and some pdfs?

File: 4.analyze-components/scripts/nbconverted/1.visualize-reconstruction.r [hunk 3]
```diff
@@ -30,10 +35,15 @@ target_vae_recon_cost_df <- compile_reconstruction_data(dataset, data_focus = "v
```
Comment (ajlee21) [inline 4.analyze-components/scripts/nbconverted/1.visualize-reconstruction.r]: Is comment here up to date?

File: 4.analyze-components/scripts/nbconverted/1.visualize-reconstruction.r [hunk 4]
```diff
@@ -30,10 +35,15 @@ target_vae_recon_cost_df <- compile_reconstruction_data(dataset, data_focus = "v
 
 target_vae_loss_gg <- plot_vae_training(target_vae_recon_cost_df)
 
-ggsave(file.path(base_dir, paste0("vae_training_reconstruction_", dataset, ".pdf")),
-       plot = target_vae_loss_gg, dpi = 500, height = 6, width = 4)
-ggsave(file.path(base_dir, paste0("vae_training_reconstruction_", dataset, ".png")),
-       plot = target_vae_loss_gg, height = 6, width = 4)
+for(extension in c('.png', '.pdf')) {
```
Comment (ajlee21) [inline 4.analyze-components/scripts/nbconverted/1.visualize-reconstruction.r]: Looks like you're doing the same thing with different datasets, is it worth it to have a for loop for this instead?

File: 4.analyze-components/scripts/nbconverted/1.visualize-reconstruction.r (hunk 2 above)
Comment (gwaygenomics) [reply 4.analyze-components/scripts/nbconverted/1.visualize-reconstruction.r]: i make both pdfs and pngs for all plots

File: 4.analyze-components/scripts/nbconverted/1.visualize-reconstruction.r (hunk 4 above)
Comment (gwaygenomics) [reply 4.analyze-components/scripts/nbconverted/1.visualize-reconstruction.r]: yeah, I think it is worth making this a function. I will update! Thanks!

File: 4.analyze-components/scripts/nbconverted/1.visualize-reconstruction.r (hunk 3 above)
Comment (gwaygenomics) [reply 4.analyze-components/scripts/nbconverted/1.visualize-reconstruction.r]: Nope, good catch

**Comment by gwaygenomics** [comment]
> Is there anyway to add a link to the bash script in the python nb so that its easier?

I am not sure what you mean. A link to which bash script?

**Pull request merged**